contrail-agent/metadata.yaml

```diff
@@ -9,6 +9,10 @@ series:
   - xenial
   - bionic
 subordinate: true
+provides:
+  nrpe-external-master:
+    interface: nrpe-external-master
+    scope: container
```

Comment:
is it OK to define scope container for subordinate charm?

Reply:
Yes. As described in the **Relations** section of the [Juju subordinate documentation page](https://docs.jujucharms.com/2.5/en/authors-subordinate-application)